Update README.md (#29)
IntelPython/scikit-learn_bench: xgboost/README.md

## How to create conda environment for benchmarking

> `conda create -n skl_bench -c conda-forge python=3.7 xgboost pandas`

Reviewer: ```suggestion

    conda create -n skl_bench -c conda-forge python=3.7 xgboost pandas
```

##  Algorithms parameters

> Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.

Reviewer: ```suggestion
You can launch benchmarks for each algorithm separately. The table below lists all supported parameters for each algorithm.
```
